README.md

```diff
@@ -1,296 +1,94 @@
+  - `├──`[`eslint-plugin-zowe-explorer`](./packages/eslint-plugin-zowe-explorer) — includes necessary files to configure ESLint plug-in for Zowe Explorer
+  - `├──`[`zowe-explorer-api`](./packages/zowe-explorer-api) — includes files to set up and use Extensibility API for Explorer. The API has two modules: Profiles API and Tree API.
+  - `├──`[`zowe-explorer-ftp-extension`](./packages/zowe-explorer-ftp-extension) — includes files to set up and use the FTP extension for Zowe Explorer.
+  - `├──`[`zowe-explorer`](./packages/zowe-explorer) — core Zowe Explorer source files, ReadMe, Changelog, and more.
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Click the **Create New Data Set** button to create a PDS.
-4. From the drop-down menu, select the type of PDS that you want to create.
-5. Enter a name for the PDS.
-   The PDS is created.
-6. To create a member, right-click the PDS and select **Create New Member**.
-7. Enter a name for the member.
-   The member is created.
+## Build Locally
 
-![Create](/docs/images/ZE-cre-pds-member.gif?raw=true "Create")
-<br /><br />
+Ensure that you meet the [software requirements](#requirements) before you build and test your Zowe Explorer.
 
-### Delete a PDS member and PDS
+Clone the repository and run `yarn run package` to build a VSIX file and start working with the extension.
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Open the profile and PDS containing the member.
-4. Right-click on the PDS member that you want to delete and select **Delete Member**.
-5. Confirm the deletion by clicking **Yes** on the drop-down menu.
+1. Clone the repository by issuing the following command:
 
-   **Note:** Alternatively, you can select 'No' to cancel the deletion.
+   ```shell
+   git clone --origin=upstream --branch=main --single-branch https://github.com/zowe/vscode-extension-for-zowe.git
+   ```
 
-6. To delete a PDS, right-click the PDS and click **Delete PDS**, then confirm the deletion.
+2. From your local copy of the repository, issue the following commands:
 
-   **Note:** You can delete a PDS before you delete its members.
+   ```shell
+   yarn install
+   yarn run package
+   ```
 
-![Delete](/docs/images/ZE-del-pds-member.gif?raw=true "Delete")
-<br /><br />
+You can find the VSIX file in the `dist` folder.
 
-### View and access multiple profiles simultaneously
+Now install the extension to VS Code.
 
-1. Navigate to the explorer tree.
-2. Open the **DATA SETS** bar.
-3. Click the **Add Profile** button on the right of the **DATA SET** explorer bar.
-4. Select the profile that you want to add to the view as illustrated by the following screen.
+1. Navigate to the Extensions menu in VS Code and click the `...` button in the top-left corner of the pane.
+2. Select Install from VSIX and select the .vsix file that was created by your `yarn workspace vscode-extension-for-zowe package` command.
```

Comment:
This step references a command that the user hasn't run according to our instructions. Perhaps we could just omit the command entirely and more generally say:

" Select Install from VSIX and select the `.vsix` file that was created by your earlier commands."

We might also want to mention to users that with our instructions, two `vsix` files are created:
- `vscode-extension-for-zowe-x.x.x.vsix` (Zowe Explorer core VS Code extension)
- `zowe-explorer-ftp-extension-x.x.x.vsix` (Zowe Explorer Extension for FTP)

Our instructions work for both VS Code extensions. Users would just need to select the `.vsix` file for the one they are interested in installing at this step.